package.json

```diff
@@ -187,5 +189,9 @@
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
+  },
+  "snyk": true,
+  "dependencies": {
+    "snyk": "^1.342.2"
```

Comment:
This should be moved to devDependencies

Reply:
Does that mean you're gonna move it or...? 😛

Reply:
I can do but left this as a heads up for whoever picks this up